src/main/java/cubicchunks/asm/mixin/core/common/MixinWorld_HeightLimits.java

```diff
@@ -191,11 +191,15 @@ private void isAreaLoadedInject(int xStart, int yStart, int zStart, int xEnd, in
                 this.testForCubes(
                         xStart, yStart, zStart,
                         xEnd, yEnd, zEnd,
-                        cube -> cube != null);
+                        this::testNotNull);
```

Comment:
it should work. What is the exception? Maybe the eclipse compiler is wrong there